src/Sequence.php

```diff
@@ -226,7 +226,7 @@ public function getGaps(): self
      */
     public function getTotalTimestampInterval(): float
```

Comment:
you revert the method name change 😄

Reply:
Yes, as I said your name makes sense if you intend the inner time between the boundaries.
However I intend the total sum of timestamp intervals.